src/Umbraco.Web.UI.Client/src/views/languages/edit.controller.js

```diff
@@ -0,0 +1,124 @@
+(function () {
+    "use strict";
+
+    function LanguagesEditController($timeout, $location, $routeParams, notificationsService, localizationService) {
+
+        var vm = this;
+
+        vm.page = {};
+        vm.language = {};
+        vm.availableLanguages = [];
+        vm.breadcrumbs = [];
+        vm.labels = {};
+        vm.initIsDefault = false;
+        vm.showDefaultLanguageInfo = false;
+
+        vm.save = save;
+        vm.back = back;
+        vm.goToPage = goToPage;
+        vm.toggleMandatory = toggleMandatory;
+        vm.toggleDefault = toggleDefault;
+
+        function init() {
+
+            // localize labels
+            var labelKeys = [
+                "treeHeaders_languages",
+                "languages_mandatoryLanguage",
+                "languages_mandatoryLanguageHelp",
+                "languages_defaultLanguage",
+                "languages_defaultLanguageHelp"
+            ];
+
+            localizationService.localizeMany(labelKeys).then(function (values) {
+                vm.labels.languages = values[0];
+                vm.labels.mandatoryLanguage = values[1];
+                vm.labels.mandatoryLanguageHelp = values[2];
+                vm.labels.defaultLanguage = values[3];
+                vm.labels.defaultLanguageHelp = values[4];
+            });
+
+            if($routeParams.create) {
+                vm.page.name = "Add language";
```

Comment:
You do loads of localization above @madsrasmussen - maybe the page name should be translated as opposed to hardcoded English string